src/components/RadioListView.js

```diff
@@ -6,17 +6,21 @@ export const RadioListView = ({ items = [], header, containerRef, empty }) => {
       { header && <div class='header'>{header}</div> }
       <div class='list' ref={containerRef}>
         {
-          items.length ? items.map(item => (
-            <div class='item' data-selectable data-title={item.title} data-selected-key={item.title} key={item.title}>
-              <div class='info'>
-                <div class='title'>{item.title}</div>
-                { item.description && <div class='description'>{item.description}</div> }
+          items.length ? items.map(item => {
+            let title
+            Array.isArray(item.title) ? title = item.title[0] : title = item.title
```

Comment:
We need to make sure `(Radio)ListView` is the reusable component, I don't suggest we introduce array to the prop title

Reply:
Yea the reason I included the `Array.isArray()` check was to make sure the component remained reusable, otherwise it was failing tests. Your feedback was great though, you made me realize I can simplify logic to actually leave `RadioListView` untouched